src/components/elements/timeline.js

```diff
@@ -51,6 +53,14 @@ const FlexWrapper = styled.div`
     position: relative;
     padding-bottom: ${(props) => (props.pb ? props.pb : '4rem')};
 `
+
+const StyledItem = styled.div`
+    padding-bottom: ${(props) => props.pb};
+
+    @media ${device.mobileL} {
+        padding-bottom: ${(props) => props.pbMobile};
```

Comment:
```suggestion
        padding-bottom: ${(props) => props.pb_mobile};
```
variables should be in snake_case.